interop/deposit_contract_testnets/README.md

```diff
@@ -0,0 +1,69 @@
+# Guidelines for Eth2 testnets using a deposit contract
+
+This document provides a set of implementation guidelines for clients, aiming to simplify the interoperability testing in Eth2 phase 0 testnets using an Eth1 validator deposit contract.
+
+## Client-operated testnets
+
+An Eth2 testnet can be identified by an address of a deployed validator deposit contract, running on the Görli network. Multiple testnets can be running at the same time and all client teams are encouraged to deploy as many contracts as they see fit. 
```

Comment:
Yes, I considered this and I didn't push it only because I wanted a simpler initial spec. We can perhaps add it later, but if there is a strong appetite for it from all clients, I'll be happy to change it now.